client/app/account/page.tsx

```diff
@@ -0,0 +1,23 @@
+'use client'
+
+import React from 'react'
+import { FormContainer } from '@/_components/layout'
+import { Button, Heading } from '@/_components/ui'
+import { AccountInfo } from './component'
+
+export default function Account() {
+  return (
+    <FormContainer>
+      <div className='flex flex-col gap-8 md:gap-6 '>
+        <Heading
+          title='Account'
+          buttonComponent={<Button variant={'outline'}>Edit</Button>}
```

Comment:
This button should have an icon?
![Screenshot 2024-03-18 at 12 55 22 PM](https://github.com/HAK2024/fairy-share/assets/66394413/8fcb4101-d2a0-41ce-bc68-416e58e6d67f)

Reply:
Added it.
33cde33c0f7eda2d7d410b0a79af665154f56b44